app/src/main/java/com/github/se/eventradar/ui/hosting/Hosting.kt

```diff
@@ -0,0 +1,281 @@
+              EventCategory.SPORTS,
+              "89298"),
+          Event(
+              "NYE2027",
+              "User3", // todo
+              LocalDateTime.now(),
+              LocalDateTime.MIN,
+              Location(83.39, 66.992, "161 makepeace avenue, n666es"),
+              "Join the Community",
+              EventTicket("regular", 0.0, 10000),
+              "valerian@joytigoel.com",
+              mutableSetOf("298jhk", "jwj8223"),
+              mutableSetOf("20982e2hk", "j1ou223e8223"),
+              EventCategory.COMMUNITY,
+              "89298"))
+
+  var selectedTabIndex by remember { mutableIntStateOf(0) }
+  val context = LocalContext.current
+
+  ConstraintLayout(modifier = Modifier.fillMaxSize().testTag("hostingScreen")) {
+    val (logo, tabs, eventList, bottomNav, buttons) = createRefs()
+    Row(
+        modifier =
+            Modifier.fillMaxWidth()
+                .constrainAs(logo) {
+                  top.linkTo(parent.top, margin = 32.dp)
+                  start.linkTo(parent.start, margin = 16.dp)
+                }
+                .testTag("logo"),
+        verticalAlignment = Alignment.CenterVertically) {
+          Image(
+              painter = painterResource(id = R.drawable.event_logo),
+              contentDescription = "Event Radar Logo",
+              modifier = Modifier.size(width = 186.dp, height = 50.dp))
+        }
+
+    TabRow(
+        selectedTabIndex = selectedTabIndex,
+        modifier =
+            Modifier.fillMaxWidth()
+                .padding(top = 8.dp)
+                .constrainAs(tabs) {
+                  top.linkTo(logo.bottom, margin = 16.dp)
+                  start.linkTo(parent.start)
+                  end.linkTo(parent.end)
+                }
+                .testTag("tabs"),
+        contentColor = MaterialTheme.colorScheme.primary) {
+          Tab(
+              selected = selectedTabIndex == 0,
+              onClick = { selectedTabIndex = 0 },
+              modifier = Modifier.testTag("myHostedEventsTab"),
+          ) {
+            Text(
+                text = "My Hosted Events",
+                style =
+                    TextStyle(
+                        fontSize = 19.sp,
+                        lineHeight = 17.sp,
+                        fontFamily = FontFamily(Font(R.font.roboto)),
+                        fontWeight = FontWeight(500),
+                        color = MaterialTheme.colorScheme.onPrimaryContainer,
+                        textAlign = TextAlign.Center,
+                        letterSpacing = 0.25.sp,
+                    ),
+                modifier = Modifier.padding(bottom = 8.dp))
+          }
+        }
+
+    if (selectedTabIndex == 0) { // events I host vs events I staff?
+      EventList(
+          mockEvents,
+          Modifier.fillMaxWidth().constrainAs(eventList) {
+            top.linkTo(tabs.bottom, margin = 8.dp)
+            start.linkTo(parent.start)
+            end.linkTo(parent.end)
+          })
```

Comment:
Reusing `EventList` is great! I would move the `EventList` composable from the `Home` file into a `component` folder. It is cleaner this way than importing stuff from Home.kt into Hosting.kt. This might also lead to cyclic dependencies which we want to avoid at all costs